src/main/resources/static/scripts/rest.js

```diff
@@ -59,6 +59,40 @@ function riotNameRestCall(sumName, successCallback, errorCallback){
         )
 }
 
+function riotDataRestCall(successCallback, errorCallback){
```

Comment:
hmmm riotData and riotMatches
matches is also kind of data. Think about better names.